TESTING.md

```diff
@@ -0,0 +1,59 @@
+# Testing 'oneview' cookbook
+The 'oneview' cookbook provides already unit testing strategy, besides having linting tools.
```

Comment:
> The 'oneview' cookbook provides already unit testing strategy, besides having linting tools.

The 'oneview' cookbook *already provides* a unit testing strategy ~, besides having~ and linting tools to ensure *its code style*. (?)
The 'oneview' cookbook provides ~already~ *a* unit testing strategy ~, besides having~ *and* linting tools. (?)